rollup-plugins/helpers.mjs

```diff
@@ -11,20 +11,19 @@
  * limitations under the License.
  */
 
-import initWabt from "wabt";
+import binaryen from "binaryen";
 import { promises as fsp, readdirSync } from "fs";
 import { resolve } from "path";
 
-const wabt = initWabt();
-
 export async function compileWat(watPath, features = []) {
   const watSource = await fsp.readFile(watPath, "utf-8");
-  const module = (await wabt).parseWat(
-    watPath,
-    watSource,
-    Object.fromEntries(features.map(flag => [flag, true]))
-  );
-  return module.toBinary({ canonicalize_lebs: true }).buffer;
+  try {
+    const module = binaryen.parseText(watSource);
+    module.setFeatures(binaryen.Features.All);
```

Comment:
Do we think it’s problematic to just always enable all features? 🤔

Reply:
This is mostly fine, but can cause some surprises. One that comes to mind is that if you use a `ref.null` and GC is enabled, the resulting module will require GC even if the input only required reference-types. Overall it would be safer to only enable the specific features required for each module.